src/components/bark/bark-typography.tsx

```diff
@@ -14,7 +14,7 @@ export function BarkH2(props: { children: React.ReactNode }) {
   );
 }
 
-export function BarkH3(props: { children: React.ReactNode }) {
+export function BarkH3(props: { children: React.ReactNode, }) {
```

Comment:
Run `make fmt` once. The ensure-pretty check is complaining about this.

https://github.com/barkbank/barkbank-pawtal/actions/runs/8897976730/job/24434099172

Reply:
Done! 1584779139fe1ad5a2c9d78a8f07aedd82f3c69d